src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * Move the Router `cache_dir` to `kernel.build_dir`
  * Deprecate the `router.cache_dir` config option
  * Add `rate_limiter` tags to rate limiter services
+ * Register TypeInfo services
```

Comment:
I don't think we've ever documented the same for other components, can probably be removed.